tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/arrays/ArrayInitializerExprTest.java

```diff
@@ -154,6 +154,20 @@ public void testFloatArrayInitWithIntExpr() {
         Assert.assertEquals(arrayValue.getFloat(2), 5.0);
     }
 
+    @Test
+    public void testInferredArrayInitWithInGrpExpr() {
+        Object[] args = {};
+        Object returns = BRunUtil.invoke(compileResult, "testInferredArrayInitWithInGrpExpr", args);
+
+        Assert.assertTrue(returns instanceof BArray);
+
+        BArray arrayValue = (BArray) returns;
+        Assert.assertEquals(arrayValue.size(), 3);
+        Assert.assertEquals(arrayValue.getInt(0), 1);
+        Assert.assertEquals(arrayValue.getInt(1), 2);
+        Assert.assertEquals(arrayValue.getBString(2).getValue(), "a");
```

Comment:
Why aren't we doing the assertions in Ballerina?

Reply:
I followed the same pattern used for other test cases in this specific test

Reply:
We can do the assertions in Ballerina for new tests we add. Eventually we will migrate most of the assertions in Java to Ballerina anyway.

Reply:
Updated a41828e66acf222373ef93deff481bde6310fd77